discordWrapper.go

```diff
@@ -45,7 +45,9 @@ func (w DiscordWrapper) RefreshRichPresenceToggl(t TogglWrapper) {
 	tags := ""
 	if len(timeEntry.tags) != 0 {
 		for _, tag := range timeEntry.tags {
-			tags += "#" + tag + " "
+			if (tag != "") {
+				tags += "#" + tag + " "
+			}
```

Comment:
This change you can just straight up remove.

The for loop won't iterate a nil slice, so `tags` remains an empty string.